src/app/containers/Navigation/index.jsx

```diff
@@ -1,49 +1,104 @@
 import React, { useContext } from 'react';
-import Navigation, {
-  NavigationUl,
-  NavigationLi,
-} from '@bbc/psammead-navigation';
+import { NavigationUl, NavigationLi } from '@bbc/psammead-navigation';
+import {
+  Dropdown,
+  DropdownUl,
+  DropdownLi,
+} from '@bbc/psammead-navigation/dropdown';
 import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import Canonical from './index.canonical';
+import Amp from './index.amp';
 
-const NavigationContainer = () => {
-  const { script, translations, navigation, service, dir } = useContext(
-    ServiceContext,
-  );
+const Navigation = () => {
+  const { platform } = useContext(RequestContext);
+  const isAmp = platform === 'amp';
 
+  const {
+    script,
+    translations,
+    navigation,
+    service,
+    dir,
+    navigationSection,
+  } = useContext(ServiceContext);
   const { currentPage, skipLinkText } = translations;
 
   if (!navigation || navigation.length === 0) {
     return null;
   }
 
-  return (
-    <Navigation
-      script={script}
-      skipLinkText={skipLinkText}
-      service={service}
-      dir={dir}
-    >
-      <NavigationUl>
+  const scrollableListItems = (
+    <NavigationUl>
+      {navigation.map((item, index) => {
+        const { title, url } = item;
+        const active = index === 0;
+
+        return (
+          <NavigationLi
+            key={title}
+            url={url}
+            script={script}
+            active={active}
+            currentPageText={currentPage}
+            service={service}
+            dir={dir}
+          >
+            {title}
+          </NavigationLi>
+        );
+      })}
+    </NavigationUl>
+  );
+
+  const dropdownId = 'dropdown-menu';
+  const dropdownListItems = (
+    <Dropdown id={dropdownId} hidden={isAmp}>
+      <DropdownUl>
```

Comment:
Look at refactoring creating the lists into a shared function for both navigations.